citesphere/src/main/java/edu/asu/diging/citesphere/core/util/model/impl/GilesUtil.java

```diff
@@ -0,0 +1,44 @@
+package edu.asu.diging.citesphere.core.util.model.impl;
```

Comment:
that class should not be in the model package. The model package contains the data model of the application, not util classe.